server/app/rest_api/rest.py

```diff
@@ -128,10 +128,53 @@ def get(self):
         return make_response((jsonify({"layout": current_app.data.layout(current_app.data.data)})))
 
 
+class AnnotationsObsAPI(Resource):
+    @swagger.doc({
+        "summary": "Fetch annotations (metadata) for all observations.",
+        "tags": ["annotations"],
+        "parameters": [{
+            "in": "query",
+            "name": "annotation-keys",
+            "type": "string",
+            "description": "comma-separated annotation keys, ex: num_genes,percent_mito"
```

Comment:
why not make this a list of strings?  that would scale better if people start doing keys that need escaping.

Reply:
good point, I made the suggestion in the requirements doc, waiting on approval